src/extension.ts

```diff
@@ -100,18 +100,24 @@ export async function activateInternal(context: vscode.ExtensionContext, perfSta
             await ext.tree.refresh(ext.attachedAccountsNode);
         });
         registerCommand('azureDatabases.refresh', async (_actionContext: IActionContext, node?: AzExtTreeItem) => await ext.tree.refresh(node));
-        registerCommand('cosmosDB.detachDatabaseAccount', async (actionContext: IActionContext, node?: AzureTreeItem) => {
-            if (!node) {
-                node = await ext.tree.showTreeItemPicker<AzureTreeItem>(cosmosDBTopLevelContextValues.map((val: string) => val += AttachedAccountSuffix), actionContext);
-            }
-            if (node instanceof MongoAccountTreeItem) {
-                if (ext.connectedMongoDB && node.fullId === ext.connectedMongoDB.parent.fullId) {
-                    setConnectedNode(undefined, codeLensProvider);
-                    await node.refresh();
+        registerCommand('cosmosDB.detachDatabaseAccount', async (_context: IActionContext & ITreeItemPickerContext, node?: AzureTreeItem) => {
```

Comment:
My bad - I didn't realize `context` was going to be a shadowed variable here. 

We typically reserve names beginning with `_` for unused variables. Can you rename `_context` to not include the underscore? (using `actionContext` is fine). Sorry for the snip-snap

Reply:
No worries. I did noticed that `_` was used rarely